core/src/main/groovy/noe/workspace/WorkspaceMultipleTomcats.groovy

```diff
@@ -16,8 +16,8 @@ class WorkspaceMultipleTomcats extends WorkspaceAbstract{
   }
 
   void createAdditionalTomcatsWithRegistrations(int numberOfAdditionalTomcats) {
-    for (int i = 2; i < numberOfAdditionalTomcats + 2; i++) {
-      // We start tests always with only one tomcat
+    // Starting from 2 as 1 tomcat has already been created
+    for (int i = 2; i <= numberOfAdditionalTomcats; i++) {
```

Comment:
This change won't work, e.g. if I want to add one more tomcat, it won't get created as 2 <= 1 isn't true.
```suggestion
    for (int i = 2; i <= numberOfAdditionalTomcats + 1; i++) {
```

Reply:
Thanks, I get what you are say and it's true, but just throwing +1 at it, is the same as not changing what we had before. The whole is going to need refactoring.

Reply:
In how many places will changes have to be made, because we change the behavior of the function?

Reply:
the behavior won't change (bugs permitting) it will create the correct number of tomcats expected as currently it doesn't do that.